decidim-core/app/packs/src/decidim/identity_selector_dialog.js

```diff
@@ -1,63 +1,33 @@
 /**
- * @deprecated since feature/redesign
- *
- * Makes the #select-identity-button to open a popup for the user to
- * select with which identity they want to perform an action.
+ * Send a request to select which identity want to use
+ * NOTE: this should not be done using javascript
```

Comment:
If it should not be done using javascript, then what we should use? The `button_to` helper?

Reply:
I think @Crashillo meant that it could be solved using a form in which the identities were marked/unmarked and do the update with a single request instead of making a request with each click on an identity

Reply:
I think it'd be better to clarify this in the comment itself.

If not I think this will be a classic problem when  instead of having to guess what the "ancient people" meant. I moved this to an issue to not block the review (#11736).